stories/41-Switch.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { boolean, radios, withKnobs } from '@storybook/addon-knobs'
+import { MLSwitch } from 'marklogic-ui-library'
+import { sizes } from 'marklogic-ui-library/ml-switch'
+
+export default {
+  component: MLSwitch,
+  title: 'Data Entry/MLSwitch',
+  decorators: [withKnobs],
+}
+
+export const Basic = () => {
+  return <MLSwitch
+    checked={boolean('Checked', true)}
```

Comment:
For the sake of not confusing the design people, I think I prefer to leave out the `value`-ish prop from the knobs, because it breaks interactivity.

Reply:
For this same reason, I'm leaving out defaultValue from now on, because storybook doesn't actually update the component when you change this, since the component isn't being recreated, just updated — and defaultValue only matters on initial creation.

Reply:
Which defaultValue are you talking about? The default for each knob?

Reply:
defaultValue is a prop that most of the input components take. It determines what the component's `state.value` (or so) is upon initial creation.